org.opentosca.container.core/src/org/opentosca/container/core/next/model/ServiceTemplateInstanceState.java

```diff
@@ -1,5 +1,5 @@
 package org.opentosca.container.core.next.model;
 
 public enum ServiceTemplateInstanceState {
-    INITIAL, CREATING, CREATED, DELETING, DELETED, ERROR
+    INITIAL, CREATING, CREATED, DELETING, DELETED, ERROR , MIGRATED
```

Comment:
Don't you also need a `MIGRATING` state?

Reply:
added